tests/e2e/docker-compose.host.yml

```diff
@@ -4,8 +4,6 @@ services:
   frontend:
     container_name: ${COMPOSE_PROJECT_NAME}-frontend
     image: nginx:stable-alpine
-    ports:
-      - "30000:30000"    # frontend http
```

Comment:
@evdokimovs why this port is removed? Do they break something? It has no impact because of `network_mode: host`, but it makes things more explicit and understandable for reader.

Reply:
@tyranron у меня docker-compose не запустится если ему скормить спеку с `network_mode: host` и одновременно указынным `ports`.

```
ERROR: for medea-frontend  "host" network_mode is incompatible with port_bindings
```

Reply:
Подтверждаю: обновился с 1.23 до 1.28 - стало ошибить.